.changes/unreleased/🎉 Feature-20230116-184305.yaml

```diff
@@ -0,0 +1,8 @@
+kind: "\U0001F389 Feature"
+body: Add sort, sorted-by flags to policy search command.
```

Comment:
This will go into customer release notes. Can you rephrase to better explain to a potential user and for our changelog what this will provide? They won't have the context of the PR so need to explain it. This message will normally be different than a commit message or PR body. 

Example: the policy search command now has the ability to return presorted data. The sorting can be customized by any available field. 


Hope that helps!